erpnext/accounts/doctype/sales_invoice/pos.py

```diff
@@ -175,6 +175,13 @@ def get_items_list(pos_profile, company):
 		if args_list:
 			cond = "and i.item_group in (%s)" % (', '.join(['%s'] * len(args_list)))
 
+	bin_join = bin_cond = ""
+	if pos_profile.get('hide_unavailable_items'):
+		bin_join = ",`tabBin` b"
+		bin_cond = "and i.item_code = b.item_code and ifnull(b.actual_qty, 0) > 0 "
```

Comment:
One could argue that _available_ items are better represented by `projected_qty`.

Reply:
In case of POS, projected_qty does not make that much sense.